pkg/kv/kvserver/metrics.go

```diff
@@ -418,6 +418,17 @@ var (
 		Measurement: "Storage",
 		Unit:        metric.Unit_COUNT,
 	}
+	// TODO DURING REVIEW: what do we call this?
+	// - storage.write-stalls to match the previous two metrics that were added in 92bc49c
```

Comment:
I would support `storage.write-stalls` as the new convention. That was also the rationale in the other commit with disk stall/slow metrics.. There's little reason to tie metric names to storage engine(s).